src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -6,6 +6,7 @@ import slick.jdbc.PostgresProfile.api._
 import uk.gov.nationalarchives.Tables.{Body, BodyRow, Consignment, ConsignmentRow, Consignmentstatus, ConsignmentstatusRow, File, Series, SeriesRow}
 import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields
 import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields.{ConsignmentFilters, StartUploadInput}
+import uk.gov.nationalarchives.tdr.api.service.FileStatusService.InProgress
```

Comment:
Should really move the Status values and type to a separate trait / class instead of using values from the `FileStatusService`.

Reply:
I've created a new `Statuses` object in the util directory